openbas-front/src/admin/components/simulations/simulation/injects/InjectIndex.tsx

```diff
@@ -96,12 +97,24 @@ const InjectIndexComponent: FunctionComponent<{ exercise: ExerciseType; injectRe
               label={t('Execution details')}
               className={classes.item}
             />
+            {
+              injectResultOverviewOutput.inject_type !== 'openbas_email' && injectResultOverviewOutput.inject_type !== 'openbas_ovh_sms' && injectResultOverviewOutput.inject_type !== 'openbas_mastodon' && injectResultOverviewOutput.inject_type !== 'openbas_http_query' && (
```

Comment:
Could we have a utility class for this condition ?
Like isNotInjectContractWithPayload(), with a better naming of course